pyblish_nuke/lib.py

```diff
@@ -278,15 +279,15 @@ def dock(window):
 
         dialog.deleteLater()
 
-        # if there are more than one main window, its a floating window
+        # If there are more than one main window, its a floating window
         if len(main_windows) > 1:
```

Comment:
Ah. So much better, don't you think? :)

And I don't mean to nitpick, and I'm not asking you change anything, but for conversation, how do you feel about this in terms of readability?

```python
if len(main_windows) > 1:
    # Then it's a floating window
    ...
    if stacked_widget.count() == 1:
        # Then it's empty and we can close it, as is native Nuke UI behaviour
        ...
```

See how the English flows alongside the Python? Just a thought. :)